src/screens/LoginScreen.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { SafeAreaView } from 'react-native';
+import { withNavigation } from 'react-navigation';
+import PropTypes from 'prop-types'
+
+import LoginForm from '../components/LoginForm';
+
+class LoginScreen extends Component {
+  render() {
+    return (
+      <SafeAreaView style={{ flex: 1 }}>
+        <LoginForm navigate={this.props.navigation.navigate}/>
```

Comment:
`this.props.navigation.navigate` > `this.props.naviagation`

- navigate
- goBack
- pop
- popToTop
...

<LoginForm {...this.props} />